src/goto-instrument/havoc_utils.cpp

```diff
@@ -16,7 +16,14 @@ Date: July 2021
 #include <util/pointer_expr.h>
 #include <util/std_code.h>
 
-#include <goto-programs/goto_program.h>
+#define IS_LOOP_HAVOC "Loop havocing instrumented by applying loop contracts."
```

Comment:
Why is new code being added here?

Reply:
They should be added in the other PR. I am going to remove these new codes from `havoc_utils.cpp` and `havoc_utils.h`.